app/src/main/java/com/example/sudoku/Win.kt

```diff
@@ -57,4 +62,16 @@ class Win : Fragment() {
         val scoreEntry = ScoreboardEntryModel("", nicknameEditText.text.toString(), score)
         storageService.writeScoreEntry(scoreEntry)
     }
+
+    private fun composeMessage(message: String) {
+        val share = Intent.createChooser(Intent().apply {
+            action = Intent.ACTION_SEND
+            putExtra(Intent.EXTRA_TEXT, message)
+            type="text/plain"
+            putExtra(Intent.EXTRA_TITLE, "My Sudoku Score")
+
+            flags = Intent.FLAG_GRANT_READ_URI_PERMISSION
+        }, null)
+        startActivity(share)
```

Comment:
Nice!

Reply:
I have just added some images to the description. Feel free to check